pkgs/by-name/st/stract/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  pkgs,
```

Comment:
```suggestion
```
Please don't call for `pkgs` in a Nixpkgs package. Just use the attribute name instead.